### DISCUSSION TITLE
Support for Arc a770 intel GPU and other Intel GPU cards.

### DISCUSSION THREAD

**Discussion by kaelsonofkrypto** [nagadomi/nunif#91] [General]
Please add support for Arc a770 intel GPU if possible and other Intel GPU cards. I wasn't able to get my intel card to be recognized in iw3. As a result i can only run off cpu with my pc that has the intel gpu. Thanks

**Comment by nagadomi** [comment]
I do not have Intel Arc.
If Intel GPU instance is available on AWS or GCP in the future,  I might be able to develop it.

**Comment by AndreaJaynegit** [comment]
Ive an A770, its got 16gb of VRAM so a lot of potential.  Do have it working with OWL3d, but wondering if there are options here too.  Works for image generation LLM etc,

**Reply by nagadomi** [reply]
Support for XPU is already implemented, but since I don't have the hardware, I'm not sure about the exact installation process.
However, it runs successfully on Intel's cloud environment(Intel Max 1100).

**Reply by nagadomi** [reply]
I added a simple document https://github.com/nagadomi/nunif/blob/master/INSTALL-xpu.md
As I mentioned, I haven't tested it myself, so I'm not sure if it works.

**Reply by AndreaJaynegit** [reply]
Well, that all installed brilliantly!  Thank you very much.  I'm coding at about 6.5FPS which is greatly improved.  Only utilizing about 6gb of VRAM so now sure if more I can use there.

**Reply by nagadomi** [reply]
Good to know it’s working.
More VRAM is used when increasing `Depth Batch Size`(`--batch-size`) and `Worker Threads`(`--max-workers`). Not sure if it improves performance.

**Reply by AndreaJaynegit** [reply]
Ok thanks for that, I'll look into what those two elements do and see if its worth trying.  Thank you for the help sincerely.

**Reply by nagadomi** [reply]
https://github.com/nagadomi/nunif/issues/77#issuecomment-2005919214 should be helpful.
